app/posts/manage-school-placements/2024-10-15-accessibility-audit.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Accessibility audit and remediation
+description: What we've done to ensure the accessibility of our service
+date: 2024-10-15
+tags:
+  - accessibility
```

Comment:
As before, to avoid strange breadcrumbs, you can add a unique key:

```suggestion
  - accessibility
eleventyComputed:
  eleventyNavigation:
    key: school-placements-accessibility-audit
```